### DISCUSSION TITLE
How to make CmdLine vertical layout like in Telescope?

### DISCUSSION THREAD

**Discussion by marchank0** [folke/noice.nvim#787] [Q&A]
How to configure layout for noice like this? This is an analog for Telescope standard vertical layout:

    ┌──────────────────────────────────────────────────┐
    │                                                  │
    │                                                  │
    │                                                  │
    │                                                  │
    │                                                  │
    │    ┌────────────────────────────────────────┐    │
    │    │                3. Result               │    │
    │    │                2. Result               │    │
    │    │cursor start -> 1. Result               │    │
    │    └────────────────────────────────────────┘    │
    │    ┌────────────────────────────────────────┐    │
    │    │                 Prompt                 │    │
    │    └────────────────────────────────────────┘    │
    │                                                  │
    └──────────────────────────────────────────────────┘

How it looks in Telescope:

<img width="1440" alt="Screenshot 2024-05-07 at 16 55 26" src="https://github.com/folke/noice.nvim/assets/1183579/af9942ed-9c5e-4c09-805e-b2052422f96e">

**Comment by ilias777** [comment]
You can try to place the cmdline_popup at the bottom:

```lua
opts = {
  views = {
    cmdline_popup = {
      position = {
        row = '80%',
        col = '50%',
      },
    },
  },
}
```

<img width="1480" alt="noice_bottom" src="https://github.com/folke/noice.nvim/assets/39212564/1c7d8b15-129f-4004-8f96-36923fdebe70">

Maybe this helps.

**Comment by tj-smith47** [comment]
This results in the following:
<img width="457" alt="Screenshot 2024-06-21 at 08 04 17" src="https://github.com/folke/noice.nvim/assets/114961514/34a48e31-5117-4a22-a295-c01ee274a872">

I'm struggling with something similar, maybe the same. I'm trying to get the results to display like the OP's in way, but specifically, I want the results when I type into the cmdline to be aligned with the input prompt without resizing as I type. Is this related, or should I open a separate issue?
